### PR TITLE
Update superagent to handle expected error responses.

### PR DESCRIPTION
**Pull request by richardpringle**
Solves #75

### CODE REVIEW

**Comment by richardpringle** [comment]
@keithamus, PTAL when you get a chance

**Comment by meeber** [comment]
LGTM!

File: lib/request.js [hunk 1]
```diff
@@ -244,6 +244,9 @@ function Test (app, method, path) {
   Request.call(this, method, path);
   this.app = app;
   this.url = typeof app === 'string' ? app + path : serverAddress(app, path);
+  this.ok(function() {
+    return true;
+  })
```
Comment (keithamus) [inline lib/request.js]: Let's get a semicolon right here 😄

**Comment by keithamus** [comment]
Minor nitpick change, but if we can get that resolved then I'll be happy with it 😄

**Comment by richardpringle** [comment]
@keithamus, agreed!

Made the fix and rebased.

**Comment by richardpringle** [comment]
@keithamus any updates?

**Comment by keithamus** [comment]
LGTM!

**Pull request merged**